features/lighthouse-visualizations/documentation

```diff
@@ -0,0 +1,17 @@
+## Lighthouse Score Visualizations (Labs feature)
+
+<img width="1400" alt="Deploy view with Lighthouse visualizations" src="https://user-images.githubusercontent.com/79875905/160019039-c3e529de-f389-42bc-a3d4-458c90d59e6a.png">
+
+If you have the Lighthouse Build Plugin installed, you can now view the Lighthouse scores for your builds on your site's Deploy page by enabling the Lighthouse Visualizations feature under Labs. 
+
+<img width="1400" alt="Labs UI" src="https://user-images.githubusercontent.com/79875905/160018999-0fde5619-15c9-4622-99a2-6ec64fc69826.png">
+
+If you have multiple audits (directories, paths, etc) defined in your build, we will display a roll-up of the average Lighthouse scores for all the current build's audits plus the results for each individual audit.
+
+<img width="1400" alt="Deploy details with multiple audit Lighthouse results" src="https://user-images.githubusercontent.com/79875905/160019057-d29dffab-49f3-4fbf-a1ac-1f314e0cd837.png">
+
+Some items of note:
+- The Lighthouse Build Plugin must be installed on your site(s) in order for these score visualizations to be displayed.
+- This Labs feature is currently only enabled at the user-level, so it will need to be enabled for each individual team member that wishes to see the Lighthouse scores displayed.
```

Comment:
[sand] We can optionally link to instructions on where to enable the Lighthouse UI for added clarity. I realize we don't currently have a docs site article on this out yet, but we could try linking to this blog post in the meantime: https://www.netlify.com/blog/2021/03/31/test-drive-netlify-beta-features-with-netlify-labs/

```suggestion
- To view Lighthouse scores in the Netlify UI, you must [enable the feature through your Netlify settings](https://www.netlify.com/blog/2021/03/31/test-drive-netlify-beta-features-with-netlify-labs/). 
- This Labs feature is currently only enabled at the user-level, so each Netlify team member must enable the feature before they can view the Lighthouse scores in the UI.
```